server/src/main/java/io/crate/protocols/http/Headers.java

```diff
@@ -67,11 +66,11 @@ public static void setKeepAlive(HttpVersion httpVersion, FullHttpResponse resp)
     public static Credentials extractCredentialsFromHttpBasicAuthHeader(String authHeaderValue) {
         if (authHeaderValue == null || authHeaderValue.isEmpty()) {
             // Empty credentials.
-            return new Credentials("", new SecureString(new char[] {}));
+            return Credentials.of("", new char[] {});
```

Comment:
Can we use a static instance instead?  e.g. `Credentials.EMPTY`

Reply:
No, I got a flaky test because of that on master. See https://github.com/crate/crate/pull/15531#issuecomment-1934843623